src/test/resources/config/application.yml

```diff
@@ -48,7 +48,6 @@ spring:
             hibernate.cache.use_second_level_cache: false
             hibernate.cache.use_query_cache: false
             hibernate.generate_statistics: true
-            hibernate.hbm2ddl.auto: validate
```

Comment:
oubox table is optional, and created if spesific configration is present. this option cause app failure on startup. (btw it missing on prod profile)